jerry-core/parser/js/js-parser-util.c

```diff
@@ -572,7 +542,9 @@ parser_emit_cbc_backward_branch (parser_context_t *context_p, /**< context */
 #ifdef PARSER_DUMP_BYTE_CODE
   if (context_p->is_show_opcodes)
   {
-    JERRY_DEBUG_MSG ("  [%3d] %s\n", (int) context_p->stack_depth, name);
+    JERRY_DEBUG_MSG ("  [%3d] %s\n",
+                     (int) context_p->stack_depth,
+                     PARSER_IS_BASIC_OPCODE (opcode) ? cbc_names[opcode] : cbc_ext_names[opcode]);
```

Comment:
Are you sure `PARSER_IS_BASIC_OPCODE (opcode)` is correct?
`opcode = (uint16_t) PARSER_GET_EXT_OPCODE (opcode);` above modifies the opcode

Reply:
Hmm, I've overlooked this. I'll take the approach of `parser_emit_cbc_forward_branch` then.